src/queue/multilevel.rs

```diff
@@ -55,6 +55,13 @@ const MAX_LEVEL0_CHANCE: f64 = 0.98;
 /// The amount that the level 0 chance is increased or decreased each time.
 const ADJUST_AMOUNT: f64 = 0.06;
 
+/// the default max number of tasks the can steal from each level queue.
+const DEFAULT_STEAL_LIMIT_PER_LEVEL: [usize; LEVEL_NUM] = [64, 16, 1];
```

Comment:
Are these general numbers? Or are they just optimized for copr?

Reply:
I think they are general numbers. In the most common workload, the queue is not very long, and the average batch-size of each steal is about 1. So 16 should be big enough.